inference-engine/tests/functional/shared_test_classes/src/single_layer/convolution_backprop_data.cpp

```diff
@@ -43,20 +46,27 @@ std::string ConvolutionBackpropDataLayerTest::getTestCaseName(testing::TestParam
 void ConvolutionBackpropDataLayerTest::SetUp() {
     convBackpropDataSpecificParams convBackpropDataParams;
     std::vector<size_t> inputShape;
+    std::vector<size_t> outputShape;
     auto netPrecision = InferenceEngine::Precision::UNSPECIFIED;
-    std::tie(convBackpropDataParams, netPrecision, inPrc, outPrc, inLayout, outLayout, inputShape, targetDevice) = this->GetParam();
+    std::tie(convBackpropDataParams, netPrecision, inPrc, outPrc, inLayout, outLayout, inputShape, outputShape, targetDevice) = this->GetParam();
     ngraph::op::PadType padType;
     InferenceEngine::SizeVector kernel, stride, dilation;
-    std::vector<ptrdiff_t> padBegin, padEnd;
+    std::vector<ptrdiff_t> padBegin, padEnd, outPadding;
     size_t convOutChannels;
-    std::tie(kernel, stride, padBegin, padEnd, dilation, convOutChannels, padType) = convBackpropDataParams;
+    std::tie(kernel, stride, padBegin, padEnd, dilation, convOutChannels, padType, outPadding) = convBackpropDataParams;
     auto ngPrc = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(netPrecision);
     auto params = ngraph::builder::makeParams(ngPrc, {inputShape});
     auto paramOuts = ngraph::helpers::convert2OutputVector(
             ngraph::helpers::castOps2Nodes<ngraph::op::Parameter>(params));
     auto convBackpropData = std::dynamic_pointer_cast<ngraph::opset1::ConvolutionBackpropData>(
             ngraph::builder::makeConvolutionBackpropData(paramOuts[0], ngPrc, kernel, stride, padBegin,
+                                                        padEnd, dilation, padType, convOutChannels, false, outPadding));
+    if (!outputShape.empty()) {
+        auto outShape = ngraph::opset3::Constant::create(ngraph::element::i64, {outputShape.size()}, outputShape);
+        convBackpropData = std::dynamic_pointer_cast<ngraph::opset1::ConvolutionBackpropData>(
+        ngraph::builder::makeConvolutionBackpropData(paramOuts[0], outShape, ngPrc, kernel, stride, padBegin,
                                                         padEnd, dilation, padType, convOutChannels));
+    }
```

Comment:
For complex initialization consider IIFE in the future. 
```suggestion
    auto convBackpropData = [&]{
        using namespace ngraph;
        if (outputShape.empty()){
            return std::dynamic_pointer_cast<opset1::ConvolutionBackpropData>(
                builder::makeConvolutionBackpropData(paramOuts[0], ngPrc, kernel, stride, padBegin,
                                            padEnd, dilation, padType, convOutChannels, false, outPadding));
        } 
        auto outShape = opset3::Constant::create(element::i64, {outputShape.size()}, outputShape);
        return std::dynamic_pointer_cast<opset1::ConvolutionBackpropData>(
            builder::makeConvolutionBackpropData(paramOuts[0], outShape, ngPrc, kernel, stride, padBegin,
                                                    padEnd, dilation, padType, convOutChannels));
        
    }();
```

Reply:
To make IIFE effective I think convBackpropData shall be const, right?
` const auto convBackpropData = [&]`